### PR TITLE
Fix C++17ism

### PR DESCRIPTION
**Pull request by ktf**

### CODE REVIEW

File: Utilities/aliceHLTwrapper/test/testMessageFormat.cxx [hunk 1]
```diff
@@ -17,7 +17,7 @@
 #include "Headers/DataHeader.h"
 #include "Headers/HeartbeatFrame.h"
 
-namespace o2::AliceHLT {
+namespace o2 { namespace AliceHLT {
```
Comment (sawenzel) [inline Utilities/aliceHLTwrapper/test/testMessageFormat.cxx]: is it more readable to open each new namespace on a new line? (not sure what the conventions say)

Comment (ktf) [inline Utilities/aliceHLTwrapper/test/testMessageFormat.cxx]: Given the tests pass, I would say it's fine. ;-) If it's not fine, we should have the clang-tidy / clang-format to spot it. For sure the current code has other instances of the same kind and for sure the current code is violating our requirements in terms of C++ features being used.

Comment (matthiasrichter) [inline Utilities/aliceHLTwrapper/test/testMessageFormat.cxx]: I think there is nothing in the conventions which applies directly to this, maybe the "One statement per loine"-recommendation, but I would also open each namespace on a separate line.

Comment (sawenzel) [inline Utilities/aliceHLTwrapper/test/testMessageFormat.cxx]: @ktf: you now that we are testing only a very limited amount of things (and clang-format is not yet tested). A test pass does hence not really mean that everything is fine.

**Comment by ktf** [comment]
Technically speaking, namespaces are not [statements](http://en.cppreference.com/w/cpp/language/statements), so my code is perfectly in line with coding guidelines and your changes are deliberate personal taste. I still think that if you think this should be the wanted behaviour:

* You should amend the coding conventions.
* You should enable the tool to enforce the change.

That said, the real issue is that `AliceHLT` should really become `alice_hlt` or simply `hlt` as we decided a while ago.

**Pull request merged**